### DISCUSSION TITLE
导入文件路径问题

### DISCUSSION THREAD

**Discussion by KingDracula** [midwayjs/midway#3522] [Q&A]
### Describe the problem（描述问题）

import { UserService } from '../service/user.service';
使用这种相对路径import是可以工作的，但是如果是使用vscode自动导入的路径本地调试启动的时候会找不到module
import { UserService } from 'src/service/user.service'
麻烦问一下这个需要怎么配置才可以支持？

### Midway Versions（Midway 版本）

midway-version@1.3.8

**Comment by KingDracula** [comment]
测试发现使用这个swc模式调试会报错midway-bin dev --ts --fast=swc
不使用这个模式就可以正常启动

**Reply by czy88840616** [reply]
vscode导入的路径不太对吧，至少得是相对路径或者配置了 alias https://midwayjs.org/docs/faq/alias_path

**Reply by KingDracula** [reply]
但是我使用这个命令进行启动的时候就没有报错midway-bin dev --ts  --fast

**Reply by czy88840616** [reply]
这种情况一般是由于配置了 tsconfig 的 root，以及和不同的编译器路径查找逻辑有关系。由于 @midwayjs/cli 不再迭代，不再推荐使用 --fast=swc 这些其他编译器的能力。

--faat 是直接走 tsc 的，应该还好。

后续我们会将 dev 切换为新的 mwtsc 工具， https://midwayjs.org/docs/tool/mwtsc，是直接基于 tsc 原生进行处理的，是最为标准的。
